src/scripts/main.js

```diff
@@ -22,7 +22,25 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key].length > 0)) {
+      const nestedEl = createTree(li, data[key]);
+
+      li.append(nestedEl);
+    }
+
+    ul.append(li);
+  }
+
+  tree.append(ul);
```

Comment:
Replace `tree.append(ul)` with `element.append(ul)` to add the created list to the given element instead of the global `tree` variable